Tools/translate.R

```diff
@@ -8,5 +8,6 @@ if (length(args)!=1) {
   rootfolder<- args[1]
 }
 
-tools::update_pkg_po(rootfolder)
+Sys.setlocale(category = "LC_ALL", locale = "en_US.UTF-8")
```

Comment:
Im guessing this was cooked up by @TimKDJ ?

Reply:
He's the culprit

Reply:
No, this is @FransMeerhoff.
It is to ensure that unicode is handled correctly on his ASCII system ;)